### PR TITLE
docs: update running instructions in README

### PR DESCRIPTION
**Pull request by hamdyelbatal122**
Update bower installation and execution instructions in README.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,7 +22,6 @@ Install Bower dependencies
 ```bash
 $ polymer install
 ```
-
```
Comment (amahdy) [inline README.md:25]: Why remove this line?

File: README.md [hunk 2]
```diff
@@ -43,6 +42,7 @@ Modifications done in the app are persisted in the brwoser, you can reset them u
 Tests can be run locally by visiting this URL: [http://localhost:8080/components/polymer-wc-app/test/](http://localhost:8080/components/polymer-wc-app/test/).
 
 You can also run tests using command line
+
```
Comment (amahdy) [inline README.md:45]: To keep consistency, command examples are not preceded by a line.

File: README.md [hunk 3]
```diff
@@ -43,6 +42,7 @@ Modifications done in the app are persisted in the brwoser, you can reset them u
...
 ```bash
 $ polymer test
-```
\ No newline at end of file
+```
```
Comment (amahdy) [inline README.md:48]: What's the change here?

**Review by amahdy — CHANGES_REQUESTED** [review]
Thanks @hamdyelbatal122 for the PR! I placed few comments, can you please take a look?

File: README.md (hunk 1 above)
Comment (hamdyelbatal122) [reply README.md]: sorry remove it by wrong ,this is great work form grea egyption

**Pull request closed without merging**